tpot/base.py

```diff
@@ -632,7 +633,9 @@ def _update_top_pipeline(self, features, target):
                     # Add an extra line of spacing if the progress bar was used
                     if self.verbosity >= 2:
                         print('')
-                    print('Best pipeline: {}'.format(self._optimized_pipeline))
+                    
+                    optimized_pipeline_str = self.clean_pipeline_string(self._optimized_pipeline)
+                    print('Best pipeline:', optimized_pipeline_str)
```

Comment:
Yes, in both [python 2](https://docs.python.org/2/library/functions.html#print) and [python 3](https://docs.python.org/3/library/functions.html#print), a blank space (' ') is the default value for the `sep` parameter which indicates how the strings should be separated in a print function.